bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2029-miot.json

```diff
@@ -0,0 +1,1076 @@
+				"friendlyName": "Audio - Voice Packet Id",
+				"channel": "voice_packet_id",
+				"type": "String",
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "STRING"
+					}
+				]
+			},
+			{
+				"property": "voice-change-state",
+				"siid": 7,
+				"piid": 3,
+				"friendlyName": "Audio - Voice Change State",
+				"channel": "voice_change_state",
+				"type": "String",
+				"stateDescription": {
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "set-voice",
+				"siid": 7,
+				"piid": 4,
+				"friendlyName": "Audio - Set Voice",
+				"channel": "set_voice",
+				"type": "String",
+				"refresh": false,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "STRING"
+					}
+				]
+			},
+			{
+				"property": "time-zone",
+				"siid": 8,
+				"piid": 1,
+				"friendlyName": "Time - Time Zone",
+				"channel": "time_zone",
+				"type": "String",
+				"stateDescription": {
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "timer-clean",
+				"siid": 8,
+				"piid": 2,
+				"friendlyName": "Time - Timer Clean",
+				"channel": "timer_clean",
+				"type": "String",
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "STRING"
+					}
+				]
+			},
+			{
+				"property": "first-clean-time",
+				"siid": 12,
+				"piid": 1,
+				"friendlyName": "Clean Logs - First Clean Time",
+				"channel": "first_clean_time",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": -1,
+					"step": 1,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "total-clean-time",
+				"siid": 12,
+				"piid": 2,
+				"friendlyName": "Clean Logs - Total Clean Time",
+				"channel": "total_clean_time",
+				"type": "Number:Time",
+				"unit": "minutes",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": -1,
+					"step": 1,
+					"pattern": "%.0f %unit%",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "total-clean-times",
+				"siid": 12,
+				"piid": 3,
+				"friendlyName": "Clean Logs - Total Clean Times",
+				"channel": "total_clean_times",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": -1,
+					"step": 1,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "total-clean-area",
+				"siid": 12,
+				"piid": 4,
+				"friendlyName": "Clean Logs - Total Clean Area",
+				"channel": "total_clean_area",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": -1,
+					"step": 1,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			}
+		],
+		"experimental": true
```

Comment:
if you have tested it with your device and it works, you can remove the line or mark it as false